easybuild/tools/options.py

```diff
@@ -177,6 +178,7 @@ def override_options(self):
             'allow-modules-tool-mismatch': ("Allow mismatch of modules tool and definition of 'module' function",
                                             None, 'store_true', False),
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
+            'compile-lmod-caches': ("Compile Lmod cache files with luac after updating them", None, 'store_true', False),
```

Comment:
`spider` just dumps the cache to `stdout`, there is no tool included in Lmod (yet?) that takes care of the `.old`, compiling, etc. I'm making the compilation optional here, since `luac` may not be around, and for older Lmod version the compiled cache is useless (since Lmod doesn't pick it up).